build/stf-run-ci/tasks/main.yml

```diff
@@ -47,6 +47,12 @@
   tags:
     - pre-clean
 
+- name: Set up bundle registry credentials (deploy from bundles)
+  ansible.builtin.include_tasks: setup_registry_auth.yml
+  when:
+  - __deploy_from_bundles_enabled | bool
+  - setup_bundle_registry_auth | bool
```

Comment:
Previously, only the auth-related block was behind this flag (L1 in setup_stf_from_bundles.yml), and there was the ability to alter `setup_bundle_registry_auth` and `setup_bundle_registry_tls_ca` separately depending on your needs based on where the bundles are hosted. 

AFAICT, both auth and tls are behind this same flag now, and setup_bundle_registry_tls_ca no longer does anything. That's probably okay, at least I don't know of any use cases offhand where we install from bundles with TLS but no auth or vice-versa, but if you're removing this functionality I think you should also change the README to reflect:

https://github.com/infrawatch/service-telemetry-operator/blob/fc28f2f77192cb81cea8fd5aa95bf2d51edcb7e8/build/stf-run-ci/README.md?plain=1#L26

https://github.com/infrawatch/service-telemetry-operator/blob/fc28f2f77192cb81cea8fd5aa95bf2d51edcb7e8/build/stf-run-ci/README.md?plain=1#L122

Reply:
That's a good point, I don't think we should alter the behavior, even though we don't have proof of this feature being used somewhere else. I will fix this in a follow up patch.